MSOfficeUpdates/MSOffice2011UpdateInfoProvider.py

```diff
@@ -43,6 +43,7 @@
 CULTURE_CODE = "0409"
 BASE_URL = "http://www.microsoft.com/mac/autoupdate/%sMSOf14.xml"
 MUNKI_UPDATE_NAME = "Office2011_update"
+URL_DL = "http"
```

Comment:
I'd suggest something like DOWNLOAD_URL_SCHEME for the name, with values of "http" or "https".

Is the update XML not retrievable via https?